docs/pages/versions/unversioned/sdk/splash-screen.mdx

```diff
@@ -83,7 +93,77 @@ export default function App() {
 
 ## Configuration
 
-To configure `expo-splash-screen`, see the following [app config](/workflow/configuration/) properties.
+You can configure `expo-splash-screen` using its built-in [config plugin](/config-plugins/introduction/) if you use config plugins in your project ([EAS Build](/build/introduction) or `npx expo run:[android|ios]`). The plugin allows you to configure various properties that cannot be set at runtime and require building a new app binary to take effect. If your app does **not** use EAS Build, then you'll need to manually configure the package.
+This is now the preferred method of configuring the splash screen. The other methods are now considered legacy and will be removed in the future.
```

Comment:
```suggestion
You can configure `expo-splash-screen` using its built-in [config plugin](/config-plugins/introduction/) if you use config plugins in your project ([EAS Build](/build/introduction) or `npx expo run:[android|ios]`). The plugin allows you to configure various properties that cannot be set at runtime and require building a new app binary to take effect. If your app does **not** use EAS Build, then you'll need to manually configure the package.

**Using the config plugin, as shown below, is the recommended method for configuring the splash screen.** The other methods are now considered legacy and will be removed in the future.
```

Let's refer to the config plugin example below and make it bold to highlight it.